frontend_app_file/src/container/File.jsx

```diff
@@ -94,10 +94,10 @@ export class File extends React.Component {
     i18n.changeLanguage(this.state.loggedUser.lang)
 
     props.registerCustomEventHandlerList([
-      { name: CUSTOM_EVENT.SHOW_APP, handler: this.handleShowApp },
-      { name: CUSTOM_EVENT.HIDE_APP, handler: this.handleHideApp },
-      { name: CUSTOM_EVENT.RELOAD_CONTENT, handler: this.handleReloadContent },
-      { name: CUSTOM_EVENT.RELOAD_APP_FEATURE_DATA, handler: this.handleReloadAppFeatureData },
+      { name: CUSTOM_EVENT.SHOW_APP(this.state.config.slug), handler: this.handleShowApp },
+      { name: CUSTOM_EVENT.HIDE_APP(this.state.config.slug), handler: this.handleHideApp },
+      { name: CUSTOM_EVENT.RELOAD_CONTENT(this.state.config.slug), handler: this.handleReloadContent },
+      { name: CUSTOM_EVENT.RELOAD_APP_FEATURE_DATA(this.state.config.slug), handler: this.handleReloadAppFeatureData },
```

Comment:
ideally, these 2 handlers can be removed.
Since app should reload it's data based on TLM, there should be no need for the custom event "reload content" and "reload app data"

Reply:
I agree for `RELOAD_APP_FEATURE_DATA`, but show app dispatch an event `RELOAD_CONTENT` if the content has changed

Reply:
mmh, I don't think that is an issue because for "show app" to work, app must exists. So the TLM content.modified should have already been handled

Reply:
When the app file is opened or a new content need to be loaded in the app file, A customEvent is sent to the app file in order to give its info. It isn't handled by any TLM because TLM only cover when a file is modified in backend (deleted, created...)

Reply:
ok